mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.erb

```diff
@@ -416,6 +440,11 @@ func expandIpv6AccessConfigs(configs []interface{}) []*compute.AccessConfig {
 				iacs[i].Name = name.(string)
 			}
 			iacs[i].Type = "DIRECT_IPV6" // Currently only type supported
+			<% unless version == 'ga' -%>
+			if sp, ok := data["security_policy"]; ok && sp != "" {
+				iacs[i].SecurityPolicy = sp.(string)
```

Comment:
As `security_policy` in the access config is computed, is this block needed?

Reply:
Removed.